specification/storage/resource-manager/Microsoft.Storage/stable/2021-09-01/common.json

```diff
@@ -88,7 +88,7 @@
         },
         "allowPermanentDelete": {
           "type": "boolean",
-          "description": "This property when set to true allows deletion of the soft deleted blob versions and snapshots. This property cannot be used blob restore policy. This property only applies to blob service and does not apply to containers or faileshare."
+          "description": "This property when set to true allows deletion of the soft deleted blob versions and snapshots. This property cannot be used blob restore policy. This property only applies to blob service and does not apply to containers or faile share."
```

Comment:
should be "file share" instead of "faile share"

Reply:
this is stale comment, it was fixed already